package.json

```diff
@@ -3,6 +3,9 @@
     "build": "tsc --build tsconfig.json && node tools/package.js",
     "lint": "eslint --ext .ts,.tsx src/"
   },
+  "exports": {
```

Comment:
Do we really need this here, if production `package.json` file is generated by script?